Makefile

```diff
@@ -41,79 +45,36 @@ fix: ## Fixup files in the repo.
 
 .PHONY: clean
 clean: ## Cleanup build artifacts and tool binaries.
-	rm -rf $(BUILD_DIR) dist tools/bin
+	rm -rf $(BUILD_DIR) dist $(TOOLS_DIR)
 
 ##@ Build
 
 .PHONY: build
-build: ## Build operator-sdk, ansible-operator, and helm-operator.
-	mkdir -p $(BUILD_DIR) && go build $(GO_BUILD_ARGS) -o $(BUILD_DIR) ./cmd/operator-sdk ./cmd/ansible-operator ./cmd/helm-operator
+build: build/operator-sdk build/ansible-operator build/helm-operator ## Build operator-sdk, ansible-operator, and helm-operator.
 
 .PHONY: install
 GOBIN ?= $(shell go env GOPATH)/bin
 install: ## Install operator-sdk, ansible-operator, and helm-operator in $GOBIN
 	go install $(GO_BUILD_ARGS) ./cmd/{operator-sdk,ansible-operator,helm-operator}
 
-.PHONY: image-build-sdk image-push-sdk image-push-sdk-multiarch
-OPERATOR_SDK_CI_IMAGE = quay.io/operator-framework/operator-sdk
-OPERATOR_SDK_IMAGE ?= $(OPERATOR_SDK_CI_IMAGE)
-OPERATOR_SDK_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-sdk: build
-	./hack/image/build-sdk-image.sh $(OPERATOR_SDK_CI_IMAGE):dev
-image-push-sdk:
-	./hack/image/push-image-tags.sh $(OPERATOR_SDK_CI_IMAGE):dev $(OPERATOR_SDK_IMAGE)-$(shell go env GOARCH)
-image-push-sdk-multiarch:
-	./hack/image/push-manifest-list.sh $(OPERATOR_SDK_IMAGE) ${OPERATOR_SDK_ARCHES}
-
-.PHONY: image-build-ansible image-push-ansible image-push-ansible-multiarch
-ANSIBLE_CI_IMAGE = quay.io/operator-framework/ansible-operator
-ANSIBLE_IMAGE ?= $(ANSIBLE_CI_IMAGE)
-ANSIBLE_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-ansible: build
-	./hack/image/build-ansible-image.sh $(ANSIBLE_CI_IMAGE):dev
-image-push-ansible:
-	./hack/image/push-image-tags.sh $(ANSIBLE_CI_IMAGE):dev $(ANSIBLE_IMAGE)-$(shell go env GOARCH)
-image-push-ansible-multiarch:
-	./hack/image/push-manifest-list.sh $(ANSIBLE_IMAGE) ${ANSIBLE_ARCHES}
-
-.PHONY: image-build-helm image-push-helm image-push-helm-multiarch
-HELM_CI_IMAGE = quay.io/operator-framework/helm-operator
-HELM_IMAGE ?= $(HELM_CI_IMAGE)
-HELM_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-helm: build
-	./hack/image/build-helm-image.sh $(HELM_CI_IMAGE):dev
-image-push-helm:
-	./hack/image/push-image-tags.sh $(HELM_CI_IMAGE):dev $(HELM_IMAGE)-$(shell go env GOARCH)
-image-push-helm-multiarch:
-	./hack/image/push-manifest-list.sh $(HELM_IMAGE) ${HELM_ARCHES}
+# Build operator-sdk project binaries
+build/scorecard-test build/scorecard-test-kuttl build/custom-scorecard-tests:
+	go build $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $@ ./images/$(@F)
+build/%:
+	go build $(GO_BUILD_ARGS) -o $@ ./cmd/$*
```

Comment:
I'm fairly sure this will be slower (2x slower on my laptop) than building all of the binaries at once. Not sure what the exact reason is, but the Go compiler seems to benefit from some shared overhead or code re-use or something when building all three at the same time.

Also, I think we need to keep the `.PHONY` declarations for these targets. I noticed that subsequent `make build` invocations do nothing even after I've changed the source code.